orion/pkg/controllers/achieve_controller.go

```diff
@@ -8,7 +8,10 @@ import (
 )
 
 func GetAllAchievements(c *gin.Context) {
-	achieveList, err := services.AchieveService.GetAll()
+	// Incoming optional parameter
+	published := c.Query("published")
```

Comment:
can we convert this to a boolean?
Basically if the value is "true" -> `true`. Otherwise `false`